{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -15,7 +16,7 @@ env:
 install: pip install -U tox
 
 # command to run tests, e.g. python setup.py test
-script: tox
+script: tox -e $TOXENV
```

Comment:
Looks like you right, -e option isn't necessary here. I found the solution in py3.5 builds in your cookiecutter's .travis.yml that uses TOX_ENV. Pushed the fix. Thank you.